### PR TITLE
feat(rules): add html-lang-require rule

### PR DESCRIPTION
**Pull request by baleyko**
* added html-lang-require rule
* add tests for html-lang-require rule

This rule helps make HTML documents to correspond to the W3C requirements.

If the rule is enabled, the next example should be valid:

```html
<html lang="en-EN"></html>
```

And the next one should not be valid:

```html
<html></html>
```

### CODE REVIEW

File: test/rules/html-lang-require.spec.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+const HTMLHint = require('../../dist/htmlhint.js').HTMLHint
+
+const ruldId = 'html-lang-require'
+const ruleOptions = {}
+
+ruleOptions[ruldId] = true
+
+describe(`Rules: ${ruldId}`, () => {
+  it('Html tag have no a lang attribute should result in an error', () => {
```
Comment (coliff) [inline test/rules/html-lang-require.spec.js:11]: `Html` here should be `HTML`

Comment (baleyko) [reply test/rules/html-lang-require.spec.js]: fixed

**Review by thedaviddias — APPROVED** [review]
Thanks @baleyko for this, ideally the doc should also be updated but it can be done in another PR. Would you mind create another ticket to update the doc accordingly to this new rule? I let you squash and merge this ticket if you want.

**Pull request merged**